brainzutils/cache.py

```diff
@@ -65,21 +60,6 @@ def init(host="localhost", port=6379, db_number=0, namespace="", ns_versions_loc
     if len(_glob_namespace) + SHA1_LENGTH > MAX_KEY_LENGTH:
```

Comment:
this check is from back when we sha1-encoded keys. We shouldn't have this check any more, but we should still check global namepace + key namespace + key is less than the max length (when setting the value)
Max key length isn't really an issue in redis if you're less than ~10k characters, so unsure if we should keep this.

Reply:
I would say let's remove it. This check is broken anyways. If want a proper working check, we will have to put set/set_many anyways.

Reply:
ok, let's just remove it then